gen/gen.go

```diff
@@ -19,13 +19,13 @@ import (
 
 // dirMetadata is the location of the x-common repository on the filesystem.
 // We're making the assumption that the x-common repository has been cloned to
-// the same parent directory as the xgo repository.
+// the same parent directory as the Exercism go track repository.
```

Comment:
Elsewhere we have Exercism Go track (vs go track here). Do we care about the Go vs go distinction?